backend/bots/models.py

```
@@ -0,0 +1,62 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+
+from gameplay.utils.fen_parser import get_default_position_list, get_starting_position_structured_fen, get_starting_structured_board_placement, get_starting_structured_castling_rights
```

Ruff (0.8.2)

`gameplay.utils.fen_parser.get_starting_position_structured_fen` imported but unused

Remove unused import: `gameplay.utils.fen_parser.get_starting_position_structured_fen`

(F401)